src/backend/lima.ts

```diff
@@ -2086,12 +2099,20 @@ class LimaKubernetesBackend extends events.EventEmitter implements K8s.Kubernete
   async stop() {
     if (this.cfg?.enabled) {
       try {
-        await this.vm.execCommand({ root: true, expectFailure: true }, '/sbin/rc-service', '--ifstarted', 'k3s', 'stop');
+        const script = 'if [ -e /etc/init.d/k3s ]; then /sbin/rc-service --ifstarted k3s stop; fi';
+
+        await this.vm.execCommand({ root: true, expectFailure: true }, '/bin/sh', '-c', script);
       } catch (ex) {
-        console.error('k3s stop failed: ', ex);
+        console.error('Failed to stop k3s while stopping kube backend: ', ex);
       }
     }
+    await this.destroy();
```

Comment:
Same question ... do we really want to destroy the VM every time we stop it?

Reply:
The VM is not being destroyed, just the k8s client. I'll rename the methods.

Reply:
I followed the link, and yes, it's jus closing server things that would be closed anyway.